### ISSUE TITLE
2.0.1: started require deprecated `pytest-runner`

### ISSUE THREAD

**Issue by kloczek** [textX/Arpeggio#116]
```console
+ /usr/bin/python3 -sBm build -w --no-isolation
* Getting build dependencies for wheel...

ERROR Missing dependencies:
        pytest-runner
```
Please read Deprecation notes from https://github.com/pytest-dev/pytest-runner

**Comment by igordejanovic** [comment]
Released a fix in [2.0.2](https://github.com/textX/Arpeggio/releases/tag/2.0.2)

**Comment by kloczek** [comment]
Thank you 👍 😄
